lightningd/gossip_msg.c

```diff
@@ -83,8 +85,8 @@ void fromwire_gossip_getchannels_entry(const u8 **pptr, size_t *max,
 	}
 }
 
-void towire_gossip_getchannels_entry(
-    u8 **pptr, const struct gossip_getchannels_entry *entry)
+void towire_gossip_getchannels_entry(u8 **pptr,
+	const struct gossip_getchannels_entry *entry)
```

Comment:
We "should" align the `const` to the `u8` on the previous line, again with tabs followed by spaces.

Reply:
done